pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -141,10 +151,15 @@ type ReconcileVirtualMachineImport struct {
 func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling VirtualMachineImport")
+	cfg, err := r.kvConfigProvider.GetConfig()
```

Comment:
Is there any reason to get it every time `Reconcile` is triggered?

Reply:
It's there just for the time being - in the validation implementation PR it won't be there anymore.